api.ts

```diff
@@ -1182,6 +1182,23 @@ export function makeObjectFormula<ParamDefsT extends ParamDefs, SchemaT extends
   }) as ObjectPackFormula<ParamDefsT, SchemaT>;
 }
 
+/**
+ * A set of options used internally by {@link makeDynamicSyncTable}
```

Comment:
Isn't this also used by developers when they add dynamic schemas to static sync tables?

Reply:
Ah yep, updated.